diff --git a/pc/popen.c b/pc/popen.c
index dff8ec9..096d9e9 100644
--- a/pc/popen.c
+++ b/pc/popen.c
@@ -232,7 +232,7 @@ os_popen(const char *command, const char *mode )
     strncpy(curmode, mode, 3); curmode[3] = '\0';
 
 #if defined(__MINGW32__)
-    current = popen(command, mode);
+    current = _popen(command, mode);
     cur = fileno(current);
     strcpy(pipes[cur].pmode, curmode);
     return(current);
@@ -279,7 +279,7 @@ os_pclose( FILE * current)
     int fd, rval;
 
 #if defined(__MINGW32__)
-    rval = pclose(current);
+    rval = _pclose(current);
     *pipes[cur].pmode = '\0';
     return rval;
 #endif
