diff --git a/configure b/configure
index 7e63394..9ccc136 100755
--- a/configure
+++ b/configure
@@ -15278,7 +15278,7 @@ case $ocaml_cc_vendor in #(
     common_cflags="-O2 -fno-strict-aliasing -fwrapv";
     internal_cflags="$cc_warnings -fno-common -fexcess-precision=standard \
 -Wvla" ;; #(
-  mingw-*-*-gcc-*) :
+  mingw-*-*-gcc-*|mingw-*-*-clang-*) :
     internal_cflags="-Wno-unused $cc_warnings \
 -fexcess-precision=standard"
     # TODO: see whether the code can be fixed to avoid -Wno-unused
